packages/clerk-js/src/ui/contexts/components/OrganizationList.ts

```diff
@@ -80,7 +80,7 @@ export const useOrganizationListContext = () => {
     afterCreateOrganizationUrl,
     skipInvitationScreen: ctx.skipInvitationScreen || false,
     hideSlug: ctx.hideSlug || false,
-    hidePersonal: ctx.hidePersonal || false,
+    hidePersonal: organizationSettings.forceOrganizationSelection || ctx.hidePersonal || false,
```

Comment:
Should we deprecate this prop once the feature releases?

Reply:
Makes sense! Let's add it to the project as a discussion topic so we don't forget. I agree that there might be cases where someone could take advantage of it, but if we have the opportunity to remove options in favor of our recommended approach that would be good 👍

Reply:
Let's say the developer currently uses `hidePersonal=false` on the application, and then enables force an org in the Dashboard. 

The expected behavior is to hide personal workspace across all org components. Especially for the after-auth flow, because if it gets displayed, then it'd lead to a bug or the user could select a personal workspace and get stuck with a pending session.